jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py

```diff
@@ -624,7 +624,7 @@ def stats_data_to_lists(self):
         self._stdev = np.array([e.stdev for e in self.db.stats_data])
         self._obs_mid_time = np.array([e.obs_mid_time for e in self.db.stats_data])
         self._stats_mean_dark_image_files = np.array([e.mean_dark_image_file for e in self.db.stats_data])
-        self._stats_numfiles = np.array([e.source_files for e in self.db.stats_data])
+        self._stats_numfiles = np.array([len(e.source_files) for e in self.db.stats_data])
```

Comment:
Was this intended to be a part of this PR?

Reply:
This one too -- this is allowed under earlier numpy versions but not later ones.  I think the original form was a mistake anyway.  The _stats_numfiles is not used anywhere, but I believe it was intended to be an array of numbers, not an array of lists of filenames.

Reply:
Awesome, thank you for clarifying. I am always cautious when seeing code changes in non-code change related PR 😄

Reply:
Same! And thanks for your careful and thorough review.